shortcodes/ucf-faq-list-shortcode.php

```diff
@@ -157,11 +157,22 @@ public static function shortcode( $atts ) {
 				'topic_class'    => 'h5',
 			), $atts, 'ucf-faq-topic-list' );
 
-			$topics = get_terms( 'topic', array(
-				'post_type' => array( 'faq' )
-			) );
+			$args = array(
+				'post_type'      => 'faq',
+				'posts_per_page' => -1
+			);
+
+			$taxonomy = 'topic';
+			$terms = get_terms( $taxonomy, $args );
+
+			foreach ( $terms as $index=>$term ) {
```

Comment:
Should the sort order by optional? Or at least allowed to be overridden to alpha?